src/fmeval/eval_algorithms/helper_models/helper_model.py

```diff
@@ -195,12 +199,9 @@ def __init__(self, model_type: str):  # pragma: no cover
         self._bertscore = hf_evaluate.load("bertscore")
         self._model_type = model_type
 
-        # Dummy call to download the model within constructor
```

Comment:
Deleting this b/c loading the model into memory during the `evaluate_sample` integ tests causes codebuild integ tests to fail (the job just hangs/crashes). Note that I never had any issues locally, even with a several year old macbook pro.